src/main/java/net/javajudd/attis/service/AWSService.java

```diff
@@ -1,155 +1,55 @@
+import software.amazon.awssdk.services.sfn.SfnClient;
+import software.amazon.awssdk.services.sfn.model.ListStateMachinesResponse;
+import software.amazon.awssdk.services.sfn.model.ListTagsForResourceRequest;
+import software.amazon.awssdk.services.sfn.model.StartExecutionRequest;
+import software.amazon.awssdk.services.sfn.model.StartExecutionResponse;
+import software.amazon.awssdk.services.sfn.model.StateMachineListItem;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static java.lang.String.format;
-import static net.javajudd.attis.utils.PasswordUtil.generatePassword;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
 
 @Service
 @Slf4j
 public class AWSService {
+    String stepFunctionArn;
 
-    @Value("${aws.dev.ami}")
-    String awsDevAmi;
-
-    @Autowired
-    MailService mailService;
-
-    public void createIamUser(Participant participant) {
-        IamClient iam = IamClient.builder()
-                .region(Region.AWS_GLOBAL)
+    public void setStepFunctionArn(String stepFunctionArn) { this.stepFunctionArn = stepFunctionArn; }
+    public Boolean isStepFunctionArnInitialized() { return stepFunctionArn != null; }
+    public List<StateMachineListItem> getStateMachines(String tagKey) {
+        SfnClient client = SfnClient.builder()
+                .region(Region.US_EAST_2)
                 .build();
 
-        IamWaiter iamWaiter = iam.waiter();
+        ListStateMachinesResponse stateMachinesResponse = client.listStateMachines();
 
-        Tag name = Tag.builder().key("Participant").value(participant.getName()).build();
-        Tag company = Tag.builder().key("Company").value(participant.getCompany()).build();
-        Tag email = Tag.builder().key("Email").value(participant.getEmail()).build();
+        return stateMachinesResponse.stateMachines().stream().filter(machine ->
+                client.listTagsForResource(ListTagsForResourceRequest.builder().resourceArn(machine.stateMachineArn()).build()).tags().stream().anyMatch(tag ->
+                        tag.key().equals(tagKey)
+                )
+        ).collect(Collectors.toList());
+    }
 
-        CreateUserRequest request = CreateUserRequest.builder()
-                .userName(participant.getInitials())
-                .tags(name, company, email)
+    public void executeStepFunction(Participant participant) {
+        SfnClient client = SfnClient.builder()
+                .region(Region.US_EAST_2)
                 .build();
 
-        CreateUserResponse response = iam.createUser(request);
+        String participantJson = new Gson().toJson(participant);
 
-        // Wait until the user is created
-        GetUserRequest userRequest = GetUserRequest.builder()
-                .userName(response.user().userName())
+        StartExecutionRequest request = StartExecutionRequest.builder()
+                .name("AttisExecution"+UUID.randomUUID().toString().substring(0,7))
```

Comment:
Thinking about how this will be listed in the step functions, I think it would be a good idea to include the initials in the name for traceability. Otherwise, if I have to debug, I will have to go through each and everyone to find the on associated with a user. So maybe AttisExecution-<initials>-<UUID>